package.json

```diff
@@ -58,13 +60,16 @@
     ]
   },
   "dependencies": {
+    "@ember-data/debug": "^3.16.0-alpha.3",
+    "@ember-data/model": "^3.16.0-alpha.3",
+    "@ember-data/store": "^3.16.0-alpha.3",
```

Comment:
@hjdivad it's the `shouldIncludeChildAddon` hook in `index.js`. I appear to not be able to give folks working links, I think because I'm part of a github beta trial for a different kind of link.